cv_next/server/api/firebaseHelper.ts

```diff
@@ -44,15 +44,18 @@ export default class FirebaseHelper {
      * Method initiates the firebase app if not initiated yet
      * @returns FirebaseApp instance
      */
-    public static getFirebaseInstance(): FirebaseApp {
+    public static getFirebaseInstance(): App {
+      
       if (
-        FirebaseHelper.firebaseAppInstance === null ||
-        FirebaseHelper.firebaseAppInstance === undefined
+        (FirebaseHelper.firebaseAppInstance === null ||
```

Comment:
Unnecessary extra brackets. Also `!` is preferable to `null` or `undefined` checks, unless explicitly needed.